### ISSUE TITLE
Error in security audit step at github actions workflow

### ISSUE THREAD

**Issue by rinshadka** [opsfolio/resource-surveillance#97]
@Lilit0x 

All the recent GitHub actions are running error at the following step:

`cargo install cargo-audit && cargo audit`

GitHub action workflow file - https://github.com/opsfolio/resource-surveillance/blob/main/.github/workflows/rust.yml

Error:

![image](https://github.com/opsfolio/resource-surveillance/assets/40381766/0e53f975-4f36-45dc-b01c-131b6ab2b955)

@shah @razakpm @avinashkurup

**Comment by Lilit0x** [comment]
@rinshadka , yes this is known, and we'll get to it in due time. The major issue is the crates that are failing the security pass are dependencies of `surveilr` dependencies, so we cannot directly update, as far I as I know.
Thank you.
